Ex10FullScript.py

```diff
@@ -0,0 +1,116 @@
+#---Part 1---
+import numpy
+import pandas
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+def ddSim(y,t0,r,K):
+    N=y[0]
+    dNdt=r*(1-N/K)*N
+    return [dNdt]
+times=(range(0,1000))    
+
+rs=[-0.1,0.1,0.4,0.8,1]
+store_rs=pandas.DataFrame({"time":times,"rl":0,"r2":0,"r3":0,"r4":0,"r5":0})
+for i in range(0,len(rs)):
+    K=100
+    N=10
+    pars1=(rs[i],K)
+    sim=spint.odeint(func=ddSim,y0=N,t=times,args=pars1)
+    store_rs.iloc[:,i]=sim[:,0]
+a=ggplot(store_rs,aes(x="time",y="r2"))
+a+geom_point()+coord_cartesian()
+
```

Comment:
To plot multiple lines on the same plot: 
a+coord_cartesian()+geom_line()+geom_line(aes(x="time",y="r2",color="10"))+geom_line(aes(x="time",y="r3",color="50"))+geom_line(aes(x="time",y="r4",color="100"))+theme_classic()+labs(x="time",y="N")